src/sqlparser/src/ast/mod.rs

```diff
@@ -742,6 +743,9 @@ impl fmt::Display for ShowObject {
             ShowObject::Table { schema } => {
                 write!(f, "TABLES{}", fmt_schema(schema))
             }
+            ShowObject::InternalTable { schema } => {
```

Comment:
Did we add to the AST tests?

Reply:
https://github.com/risingwavelabs/risingwave/blob/main/src/sqlparser/tests/testdata/show.yaml

Reply:
I think we can skip e2e test since it may be brittle, internal table may have a non-deterministic name...